src/helper_gitlab_modify.sh

```diff
@@ -664,7 +683,7 @@ commit_changes_to_gitlab() {
 
 
 #######################################
-# Checks if repository exists in the GitLab server and creates a new empty repository 
+# Checks if repository exists in the GitLab server and creates a new empty repository
```

Comment:
I think the accuracy of this description can be increased.

Reply:
I agree, the new update should include an improved version.